src/lumigo_tracer/wrappers/sql/sqlalchemy_wrapper.py

```diff
@@ -73,16 +73,14 @@ def _handle_error(context):  # type: ignore[no-untyped-def]
 def execute_wrapper(func, instance, args, kwargs):  # type: ignore[no-untyped-def]
     result = func(*args, **kwargs)
     with lumigo_safe_execute("sqlalchemy: listen to engine"):
-        listen(result, "before_cursor_execute", _before_cursor_execute)
-        listen(result, "after_cursor_execute", _after_cursor_execute)
-        listen(result, "handle_error", _handle_error)
+        listen(instance, "before_cursor_execute", _before_cursor_execute)
+        listen(instance, "after_cursor_execute", _after_cursor_execute)
+        listen(instance, "handle_error", _handle_error)
     return result
```

Comment:
is it still correct to return `result`?

Reply:
yes :)
(Though results should always be None...)